README.md

```diff
@@ -1 +1,126 @@
-# PADIVAR-Hardware
+# PADIVAR
+
+**PADIVAR** is an open-source OBD2 vehicle diagnostic software compatible with multiple adapters. 
+It is hosted on a server connected to one (or multiple) cars, via one (or multiple) OBD2 devices, and can be accessed simultaneously by multiple clients remotely thanks to a simple TCP client.
+
+Developed by [ESIEA](https://www.esiea.fr/) students for [Le Mans Métropole](https://www.lemansmetropole.fr/).
+
+
+## Summary
+
+<!--ts-->
+   * [PADIVAR](#padivar)
+   * [Summary](#summary)
+   * [Installation](#installation)
+      * [Build](#build)
+   * [Usage](#utilisation)
+	   * [Setup](#setup) 
+		   * [Compatible OBD2 devices](#compatible-obd2-devices)
+		   * [Available commands](#available-commands)
+	   * [Connection](#connection)
+   * [License](#license)
+<!--te-->
+
+
+# Installation
+
+## Build
+
+* Download `g++` and `make`
+* Download `libbluetooth-dev` (Debian) or `libbluez3` (Mageia)
```

Comment:
is it sure for Mageia?

Reply:
Compiled on Mageia 8 with lib64bluez-devel, works fine.